helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -53,7 +53,7 @@ spec:
         sidecar.istio.io/inject: "false"
         {{- end }}
     spec:
-      {{- if not .Values.isWindows }}
+      {{- if and (.Values.hostNetwork) (not .Values.isWindows)}}
```

Comment:
If this is only applicable for the agent, I think it should be in the `agent` value mapping.

Reply:
+1

Reply:
moved to agent 😊